lib/spec/simpleofflinecache.sw-spec.js

```diff
@@ -0,0 +1,104 @@
+importScripts('/base/lib/spec/nodeMock.js');
+importScripts('/base/lib/simpleofflinecache.js');
+
+describe('SimpleOfflineCache instance', function () {
+  'use strict';
+
+  self.cacheHelper = {
+    getCache: caches.open.bind(caches),
+    fetchAndCache: function (request, cache) {
+      var response = new Response('network response');
+      cache.put(request.clone(), response.clone());
+      return response;
+    }
+  };
+
+  beforeEach(function (done) {
+    caches.keys().then(function (names) {
+      Promise.all(names.map(caches.delete.bind(caches)))
+      .then(function () { done(); })
```

Comment:
Sorry, delete is passing nothing. What is passing something (an array of `undefined`) is the `Promise.all()`.